client/gulpfile.js

```diff
@@ -6,7 +6,6 @@ const gulpStylelint = require('gulp-stylelint');
 const sourcemaps = require('gulp-sourcemaps');
 const gutil = require('gulp-util');
 const webpackConfig = require('./webpack.config.js');
-const webpackConfigHead = require('./webpack-head.config.js');
```

Comment:
Was mucking around with running the javascript in the head through babel etc. a while back and it got left in